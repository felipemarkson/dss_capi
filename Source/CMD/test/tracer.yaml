name: tracer
time_delta: 1s
broker: tcp://localhost:5570
values:
    output:
        topic: opendss/output
        default: 0

