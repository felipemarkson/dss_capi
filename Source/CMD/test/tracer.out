#time	topic	value
2	output	redirect IEEE13Base.dss
1802	output	solve
5402	output	export y triplet base_ysparse.csv
16202	output	export ynodelist base_nodelist.csv
19802	output	export summary base_summary.csv
21402	output	help
21502	output	quit
